vmdb/db/migrate/20150630025128_create_tenant.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTenant < ActiveRecord::Migration
+  def change
+    create_table :tenants do |t|
+      t.string :domain
+      t.string :subdomain
+      t.string :customer_name
+      t.string :vmdb_name
```

Comment:
<img width="233" alt="manageiq configuration 2015-07-06 11-47-23" src="https://cloud.githubusercontent.com/assets/1930/8526661/0b18a970-23d6-11e5-806a-87749e21bba1.png">


They are loaded into the session in two different places:

1. [read out of vmdb.yml] and stored into the session.
2. [read out of the configuration form] and stored in the session.

I'm hesitant to change the name of these variables too much, but we can do that.


[read out of vmdb.yml]: https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/controllers/application_controller.rb#L2357-L2358
[read out of the configuration form]: https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/controllers/ops_controller/settings/common.rb#L410-L411